cat-launcher/src-tauri/src/game_release/utils.rs

```diff
@@ -42,6 +43,28 @@ pub enum GetReleaseError {
     NotFound(String),
 }
 
+pub fn gh_release_to_game_release(
+    gh_release: &GitHubRelease,
+    variant: &GameVariant,
+    all_gh_releases: &[GitHubRelease],
+) -> GameRelease {
+    let latest_release = all_gh_releases.iter().max_by_key(|r| r.created_at);
```

Comment:
Don't calculate latest release here. Latest release should be determined in the frontend.